### ISSUE TITLE
Building (also a completely new) React based jhipster application is failing

### ISSUE THREAD

**Issue by OmarHawk** [jhipster/generator-jhipster#17614]
##### **Overview of the issue**

At the moment, an application that is completely newly set up using React as a frontend framework fails to build due to some changes in underlying libraries. This is the output with error that is produced during compilation:

```
[INFO] --- frontend-maven-plugin:1.12.0:npm (webapp build dev) @ demo-2 ---
[INFO] npm not inheriting proxy config from Maven
[INFO] Running 'npm run webapp:build' in C:\Users\jwedding\git\markant\demo2
[INFO]
[INFO] > demo-2@0.0.1-SNAPSHOT webapp:build
[INFO] > npm run clean-www && npm run webapp:build:dev --
[INFO]
[INFO]
[INFO] > demo-2@0.0.1-SNAPSHOT clean-www
[INFO] > rimraf target/classes/static/app/{src,target/}
[INFO]
[INFO]
[INFO] > demo-2@0.0.1-SNAPSHOT webapp:build:dev
[INFO] > webpack --config webpack/webpack.dev.js --env stats=minimal
[INFO]
Webpack: Starting ...
[INFO]
[INFO]   â?? Compile modules
[INFO]   â?? Build modules
[INFO]   â?? Optimize modules
[INFO]   â?? Emit files
[INFO]
[INFO] Finished after 26.118 seconds.
[INFO]
Webpack: Starting ...
[INFO]
[INFO]   â?? Compile modules
[INFO]   â?? Build modules
[INFO]   â?? Optimize modules
[INFO]   â?? Emit files
[INFO]
[INFO] Finished after 3.551 seconds.
[INFO]
[INFO] 47 assets
[INFO] 670 modules
[INFO]
[INFO] ERROR in src/main/webapp/app/shared/auth/private-route.spec.tsx:47:15
[INFO] TS2769: No overload matches this call.
[INFO]   Overload 1 of 2, '(props: RouterProps | Readonly<RouterProps>): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is missing the following properties from type 'History<unknown>': length, goBack, goForward
[INFO]   Overload 2 of 2, '(props: RouterProps, context: any): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]     45 |     const history = createMemoryHistory();
[INFO]     46 |     const { container } = wrapper(
[INFO]   > 47 |       <Router history={history}>
[INFO]        |               ^^^^^^^
[INFO]     48 |         <PrivateRouteComponent component={TestComp} path="/" />
[INFO]     49 |       </Router>,
[INFO]     50 |       {
[INFO]
[INFO] ERROR in src/main/webapp/app/shared/auth/private-route.spec.tsx:66:15
[INFO] TS2769: No overload matches this call.
[INFO]   Overload 1 of 2, '(props: RouterProps | Readonly<RouterProps>): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]   Overload 2 of 2, '(props: RouterProps, context: any): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]     64 |     const history = createMemoryHistory();
[INFO]     65 |     const { container } = wrapper(
[INFO]   > 66 |       <Router history={history}>
[INFO]        |               ^^^^^^^
[INFO]     67 |         <PrivateRouteComponent component={TestComp} path="/" />
[INFO]     68 |       </Router>,
[INFO]     69 |       {
[INFO]
[INFO] ERROR in src/main/webapp/app/shared/auth/private-route.spec.tsx:83:15
[INFO] TS2769: No overload matches this call.
[INFO]   Overload 1 of 2, '(props: RouterProps | Readonly<RouterProps>): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]   Overload 2 of 2, '(props: RouterProps, context: any): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]     81 |     const history = createMemoryHistory();
[INFO]     82 |     const { container } = wrapper(
[INFO]   > 83 |       <Router history={history}>
[INFO]        |               ^^^^^^^
[INFO]     84 |         <PrivateRouteComponent exact component={TestComp} path="/" />
[INFO]     85 |       </Router>,
[INFO]     86 |       {
[INFO]
[INFO] ERROR in src/main/webapp/app/shared/error/error-boundary-route.spec.tsx:27:15
[INFO] TS2769: No overload matches this call.
[INFO]   Overload 1 of 2, '(props: RouterProps | Readonly<RouterProps>): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]   Overload 2 of 2, '(props: RouterProps, context: any): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]     25 |     const history = createMemoryHistory();
[INFO]     26 |     const { container } = render(
[INFO]   > 27 |       <Router history={history}>
[INFO]        |               ^^^^^^^
[INFO]     28 |         <ErrorBoundaryRoute component={ErrorComp} path="/" />
[INFO]     29 |       </Router>
[INFO]     30 |     );
[INFO]
[INFO] ERROR in src/main/webapp/app/shared/layout/header/header.spec.tsx:42:19
[INFO] TS2769: No overload matches this call.
[INFO]   Overload 1 of 2, '(props: RouterProps | Readonly<RouterProps>): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]   Overload 2 of 2, '(props: RouterProps, context: any): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]     40 |       const { container } = render(
[INFO]     41 |         <Provider store={store}>
[INFO]   > 42 |           <Router history={history}>
[INFO]        |                   ^^^^^^^
[INFO]     43 |             <Header {...props} />
[INFO]     44 |           </Router>
[INFO]     45 |         </Provider>
[INFO]
[INFO] ERROR in src/main/webapp/app/shared/layout/menus/account.spec.tsx:15:17
[INFO] TS2769: No overload matches this call.
[INFO]   Overload 1 of 2, '(props: RouterProps | Readonly<RouterProps>): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]   Overload 2 of 2, '(props: RouterProps, context: any): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]     13 |       const history = createMemoryHistory();
[INFO]     14 |       const { container } = render(
[INFO]   > 15 |         <Router history={history}>
[INFO]        |                 ^^^^^^^
[INFO]     16 |           <AccountMenu isAuthenticated />
[INFO]     17 |         </Router>
[INFO]     18 |       );
[INFO]
[INFO] ERROR in src/main/webapp/app/shared/layout/menus/account.spec.tsx:27:17
[INFO] TS2769: No overload matches this call.
[INFO]   Overload 1 of 2, '(props: RouterProps | Readonly<RouterProps>): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]   Overload 2 of 2, '(props: RouterProps, context: any): Router', gave the following error.
[INFO]     Type 'MemoryHistory' is not assignable to type 'History<unknown>'.
[INFO]     25 |       const history = createMemoryHistory();
[INFO]     26 |       const { container } = (mountedWrapper = render(
[INFO]   > 27 |         <Router history={history}>
[INFO]        |                 ^^^^^^^
[INFO]     28 |           <AccountMenu />
[INFO]     29 |         </Router>
[INFO]     30 |       ));
[INFO]
[INFO] webpack 5.65.0 compiled with 7 errors in 26111 ms
```

##### **Motivation for or Use Case**

Build is failing.

##### **Reproduce the error**

Generate an application with React
Build via mvnw  (or whatever triggers the webpack build).

##### **Related issues**

Nope. Didn't find any

##### **JHipster Version(s)**
7.5.0

##### **JHipster configuration**

C:\Users\jwedding\git\markant\demo2>jhipster info
INFO! Using JHipster version installed locally in current project's node_modules
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
demo-2@0.0.1-SNAPSHOT C:\Users\jwedding\git\markant\demo2
`-- (empty)

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "applicationType": "monolith",
    "baseName": "demo2",
    "jhipsterVersion": "7.5.0",
    "skipUserManagement": false,
    "skipCheckLengthOfIdentifier": false,
    "skipFakeData": false,
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "testFrameworks": [],
    "blueprints": [],
    "otherModules": [],
    "pages": [],
    "creationTimestamp": 1642609580922,
    "serviceDiscoveryType": "no",
    "reactive": false,
    "authenticationType": "jwt",
    "packageName": "com.mycompany.myapp",
    "serverPort": "8080",
    "cacheProvider": "ehcache",
    "enableHibernateCache": true,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "postgresql",
    "buildTool": "maven",
    "serverSideOptions": [],
    "websocket": false,
    "searchEngine": false,
    "messageBroker": false,
    "enableSwaggerCodegen": false,
    "clientFramework": "react",
    "withAdminUi": true,
    "clientTheme": "none",
    "enableTranslation": true,
    "nativeLanguage": "de",
    "jwtSecretKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons",
    "devServerPort": 9060,
    "clientPackageManager": "npm",
    "clientThemeVariant": "",
    "languages": ["de", "en"],
    "enableGradleEnterprise": false
  }
}

</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>

##### **Environment and Tools**

openjdk version "11.0.12" 2021-07-20 LTS
OpenJDK Runtime Environment 18.9 (build 11.0.12+7-LTS)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.12+7-LTS, mixed mode)

git version 2.34.1.windows.1

node: v14.18.2

npm: 6.14.15

Docker version 20.10.8, build 3967b7d

docker-compose version 1.29.2, build 5becea4c

No change to package.json was detected. No package manager install will be executed.

##### **Browsers and Operating System**

Windows 10
- [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by idexxt2** [comment]
I confirmed (ran into this last night) that this also happens on the latest patched version of MacOS Monterey on an M1 MacBook Pro using several versions of Node/NPM (14.18.3, 16.13.1, 16.13.2).  I also experienced it on Mint Linux 20.3 and on a Windows 10 PC under WSL2 with Ubuntu.  I was able to make a freshly generated (and broken as described above) application work by comparing an existing app's package-lock.json to the newly generated one, and patching the new app's package-lock.json by copying in dependency sections from the old app for these dependencies: node_modules/history, @types/history, @types/react-router, history, react-router, node_modules/react-router-dom, react-router-dom, webpack-dev-server - totally unscientific hack to determine if my problem was one of the node packages without diving into a typescript debugging rabbit hole.

**Comment by qmonmert** [comment]
thanks @OmarHawk and @idexxt2 can you provide your .yo-rc.json file?

**Comment by idexxt2** [comment]
`{
  "generator-jhipster": {
    "applicationType": "monolith",
    "baseName": "BrokenNodeTest",
    "jhipsterVersion": "7.5.0",
    "skipUserManagement": false,
    "skipCheckLengthOfIdentifier": false,
    "skipFakeData": false,
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "testFrameworks": [],
    "blueprints": [],
    "otherModules": [],
    "pages": [],
    "creationTimestamp": 1642608884078,
    "serviceDiscoveryType": "no",
    "reactive": true,
    "authenticationType": "jwt",
    "packageName": "com.mindovermodel.brokennodetest",
    "serverPort": "8080",
    "cacheProvider": "no",
    "enableHibernateCache": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "postgresql",
    "buildTool": "maven",
    "serverSideOptions": [],
    "websocket": false,
    "searchEngine": false,
    "messageBroker": false,
    "enableSwaggerCodegen": false,
    "clientFramework": "react",
    "withAdminUi": true,
    "clientTheme": "cyborg",
    "clientThemeVariant": "primary",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "jwtSecretKey": "ZTk2MzJjODU5ZjViYzQ5MzBiNDNkZjk1NDcxOTdiZGQzYjc2MWVmZjZiNWJkODI3YjhmNTBhMjc2NmI2NThlMzFhMWZlM2ZjZjczYTgwZjdkNDExMDZkNDRlY2I5MDJlOTJjZTQ0YTE4MWRlZmZhMmRhNDMwYWM1MzM3MDRhMWY=",
    "devServerPort": 9060,
    "clientPackageManager": "npm",
    "languages": ["en", "al", "ar-ly", "hy"],
    "enableGradleEnterprise": false
  }
}`

**Comment by qmonmert** [comment]
thanks

**Comment by DanielFran** [comment]
We have actually same issue in "main" branch, it started yesterday I believe...

**Comment by mshima** [comment]
I think was fixed by https://github.com/jhipster/generator-jhipster/pull/17607.

**Comment by qmonmert** [comment]
@idexxt2 can you try to remove you node_modules and package-lock.json and start `npm install` with this package.json

```
{
  "name": "broken-node-test",
  "version": "0.0.1-SNAPSHOT",
  "private": true,
  "description": "Description for BrokenNodeTest",
  "license": "UNLICENSED",
  "scripts": {
    "app:start": "./mvnw",
    "backend:build-cache": "./mvnw dependency:go-offline",
    "backend:debug": "./mvnw -Dspring-boot.run.jvmArguments=\"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8000\"",
    "backend:doc:test": "./mvnw -ntp javadoc:javadoc --batch-mode",
    "backend:info": "./mvnw -ntp enforcer:display-info --batch-mode",
    "backend:nohttp:test": "./mvnw -ntp checkstyle:check --batch-mode",
    "backend:start": "./mvnw -Dskip.installnodenpm -Dskip.npm",
    "backend:unit:test": "./mvnw -ntp -Dskip.installnodenpm -Dskip.npm verify --batch-mode -Dlogging.level.ROOT=OFF -Dlogging.level.org.zalando=OFF -Dlogging.level.tech.jhipster=OFF -Dlogging.level.com.mindovermodel.brokennodetest=OFF -Dlogging.level.org.springframework=OFF -Dlogging.level.org.springframework.web=OFF -Dlogging.level.org.springframework.security=OFF",
    "build": "npm run webapp:prod --",
    "build-watch": "concurrently 'npm run webapp:build:dev -- --watch' npm:backend:start",
    "ci:backend:test": "npm run backend:info && npm run backend:doc:test && npm run backend:nohttp:test && npm run backend:unit:test -- -P$npm_package_config_default_environment",
    "ci:e2e:package": "npm run java:$npm_package_config_packaging:$npm_package_config_default_environment -- -Pe2e -Denforcer.skip=true",
    "ci:e2e:prepare": "npm run ci:e2e:prepare:docker",
    "ci:e2e:prepare:docker": "npm run docker:db:up && npm run docker:others:up && docker ps -a",
    "preci:e2e:server:start": "npm run docker:db:await --if-present && npm run docker:others:await --if-present",
    "ci:e2e:server:start": "java -jar target/e2e.$npm_package_config_packaging --spring.profiles.active=$npm_package_config_default_environment -Dlogging.level.ROOT=OFF -Dlogging.level.org.zalando=OFF -Dlogging.level.tech.jhipster=OFF -Dlogging.level.com.mindovermodel.brokennodetest=OFF -Dlogging.level.org.springframework=OFF -Dlogging.level.org.springframework.web=OFF -Dlogging.level.org.springframework.security=OFF --logging.level.org.springframework.web=ERROR",
    "ci:e2e:teardown": "npm run ci:e2e:teardown:docker",
    "ci:e2e:teardown:docker": "npm run docker:db:down --if-present && npm run docker:others:down && docker ps -a",
    "ci:frontend:test": "npm run webapp:build:$npm_package_config_default_environment && npm run test-ci",
    "clean-www": "rimraf target/classes/static/app/{src,target/}",
    "cleanup": "rimraf target/classes/static/",
    "docker:app:up": "docker-compose -f src/main/docker/app.yml up -d",
    "docker:db:down": "docker-compose -f src/main/docker/postgresql.yml down -v --remove-orphans",
    "docker:db:up": "docker-compose -f src/main/docker/postgresql.yml up -d",
    "docker:others:await": "",
    "docker:others:down": "",
    "predocker:others:up": "",
    "docker:others:up": "",
    "java:docker": "./mvnw -ntp verify -DskipTests -Pprod jib:dockerBuild",
    "java:docker:arm64": "npm run java:docker -- -Djib-maven-plugin.architecture=arm64",
    "java:docker:dev": "npm run java:docker -- -Pdev,webapp",
    "java:docker:prod": "npm run java:docker -- -Pprod",
    "java:jar": "./mvnw -ntp verify -DskipTests --batch-mode",
    "java:jar:dev": "npm run java:jar -- -Pdev,webapp",
    "java:jar:prod": "npm run java:jar -- -Pprod",
    "java:war": "./mvnw -ntp verify -DskipTests --batch-mode -Pwar",
    "java:war:dev": "npm run java:war -- -Pdev,webapp",
    "java:war:prod": "npm run java:war -- -Pprod",
    "jest": "jest --coverage --logHeapUsage --maxWorkers=2 --config jest.conf.js",
    "jest:update": "npm run jest -- --updateSnapshot",
    "lint": "eslint . --ext .js,.ts,.jsx,.tsx",
    "lint:fix": "npm run lint -- --fix",
    "prepare": "husky install",
    "prettier:check": "prettier --check \"{,src/**/,webpack/}*.{md,json,yml,html,cjs,mjs,js,ts,tsx,css,scss,java}\"",
    "prettier:format": "prettier --write \"{,src/**/,webpack/}*.{md,json,yml,html,cjs,mjs,js,ts,tsx,css,scss,java}\"",
    "start": "npm run webapp:dev --",
    "start-tls": "npm run webapp:dev -- --env.tls",
    "pretest": "npm run lint",
    "test": "npm run jest --",
    "test-ci": "npm run lint && npm run jest:update --",
    "test:watch": "npm run jest -- --watch",
    "webapp:build": "npm run clean-www && npm run webapp:build:dev --",
    "webapp:build:dev": "webpack --config webpack/webpack.dev.js --env stats=minimal",
    "webapp:build:prod": "webpack --config webpack/webpack.prod.js --progress=profile",
    "webapp:dev": "npm run webpack-dev-server -- --config webpack/webpack.dev.js --env stats=minimal",
    "webapp:dev-verbose": "npm run webpack-dev-server -- --config webpack/webpack.dev.js --progress=profile --env stats=normal",
    "webapp:prod": "npm run clean-www && npm run webapp:build:prod --",
    "webapp:test": "npm run test --",
    "webpack-dev-server": "webpack serve"
  },
  "config": {
    "backend_port": "8080",
    "default_environment": "prod",
    "packaging": "jar"
  },
  "dependencies": {
    "@fortawesome/fontawesome-svg-core": "1.2.36",
    "@fortawesome/free-solid-svg-icons": "5.15.4",
    "@fortawesome/react-fontawesome": "0.1.16",
    "@reduxjs/toolkit": "1.7.1",
    "axios": "0.25.0",
    "bootstrap": "5.1.3",
    "bootswatch": "5.1.3",
    "dayjs": "1.10.7",
    "lodash": "4.17.21",
    "path-browserify": "1.0.1",
    "react": "17.0.2",
    "react-dom": "17.0.2",
    "react-hook-form": "7.24.0",
    "react-jhipster": "0.18.0",
    "react-loadable": "5.5.0",
    "react-redux": "7.2.6",
    "react-redux-loading-bar": "5.0.2",
    "react-router-dom": "5.3.0",
    "react-toastify": "8.1.0",
    "react-transition-group": "4.4.2",
    "reactstrap": "9.0.1",
    "redux": "4.1.2",
    "redux-thunk": "2.4.1",
    "tslib": "2.3.1",
    "uuid": "8.3.2"
  },
  "devDependencies": {
    "@testing-library/react": "12.1.2",
    "@types/jest": "27.4.0",
    "@types/lodash": "4.14.178",
    "@types/node": "16.11.21",
    "@types/react": "17.0.38",
    "@types/react-dom": "17.0.11",
    "@types/react-redux": "7.1.22",
    "@types/react-router-dom": "5.3.3",
    "@types/redux": "3.6.31",
    "@types/webpack-env": "1.16.3",
    "@typescript-eslint/eslint-plugin": "5.10.0",
    "@typescript-eslint/parser": "5.10.0",
    "autoprefixer": "10.4.2",
    "browser-sync": "2.27.7",
    "browser-sync-webpack-plugin": "2.3.0",
    "concurrently": "7.0.0",
    "copy-webpack-plugin": "10.2.0",
    "core-js": "3.20.3",
    "cross-env": "7.0.3",
    "css-loader": "6.5.1",
    "css-minimizer-webpack-plugin": "3.4.1",
    "eslint": "8.7.0",
    "eslint-config-prettier": "8.3.0",
    "eslint-plugin-react": "7.28.0",
    "eslint-webpack-plugin": "3.1.1",
    "folder-hash": "4.0.1",
    "fork-ts-checker-webpack-plugin": "6.5.0",
    "generator-jhipster": "7.5.0",
    "html-webpack-plugin": "5.5.0",
    "husky": "7.0.4",
    "identity-obj-proxy": "3.0.0",
    "jest": "27.4.7",
    "jest-junit": "13.0.0",
    "jest-sonar-reporter": "2.0.0",
    "json-loader": "0.5.7",
    "lint-staged": "12.1.7",
    "merge-jsons-webpack-plugin": "2.0.1",
    "mini-css-extract-plugin": "2.5.2",
    "postcss-loader": "6.2.1",
    "postcss-rtlcss": "3.5.1",
    "prettier": "2.5.1",
    "prettier-plugin-java": "1.6.1",
    "prettier-plugin-packagejson": "2.2.15",
    "react-infinite-scroll-component": "6.1.0",
    "redux-mock-store": "1.5.4",
    "rimraf": "3.0.2",
    "sass": "1.49.0",
    "sass-loader": "12.4.0",
    "simple-progress-webpack-plugin": "2.0.0",
    "sinon": "12.0.1",
    "source-map-loader": "3.0.1",
    "sourcemap-istanbul-instrumenter-loader": "0.2.0",
    "style-loader": "3.3.1",
    "swagger-ui-dist": "4.1.3",
    "terser-webpack-plugin": "5.3.0",
    "thread-loader": "3.0.4",
    "ts-jest": "27.1.3",
    "ts-loader": "9.2.6",
    "typescript": "4.5.4",
    "wait-on": "6.0.0",
    "webpack": "5.66.0",
    "webpack-cli": "4.9.1",
    "webpack-dev-server": "4.7.3",
    "webpack-merge": "5.8.0",
    "webpack-notifier": "1.15.0",
    "workbox-webpack-plugin": "6.4.2"
  },
  "engines": {
    "node": ">=16.13.1"
  },
  "cacheDirectories": [
    "node_modules"
  ],
  "jestSonar": {
    "reportPath": "target/test-results/jest",
    "reportFile": "TESTS-results-sonar.xml"
  }
}

```

**Comment by qmonmert** [comment]
@mshima yes it works in local, I re-started build ant it seems working

**Comment by idexxt2** [comment]
@qmonmert  - yes, removing my node_modules directory and package-lock.json and using  package.json file provided above makes the app work.

**Comment by OmarHawk** [comment]
> I think was fixed by #17607.

Yes. Just increaing that version number and building freshly works.

**Comment by OmarHawk** [comment]
Closed - as this was fixed with the last release.

**Comment by rhl07** [comment]
> > I think was fixed by #17607.
> 
> Yes. Just increasing that version number in my package.json and building freshly works.

which is command you have used to build ?
